### ISSUE TITLE
behavior of indicator menu when the panel automatically minimizes

### ISSUE THREAD

**Issue by dln949** [nitroshare/nitroshare-desktop#33]
Very nice tool you have created!  

I have nitroshare 0.3.0 installed on a 32 bit machine running the 32 bit, lxde version of lubuntu 14.10.  The panel is at the bottom of the screen.  I have the panel set to minimize when not being used.  

For other indicators in the system tray area of the panel, if I bring up the popup menu for that indicator, the focus remains on the panel while I'm using the indicator's popup menu with the pointer, so that the panel does not minimize and cause the indicator's popup menu to disappear.  However, with nitroshare, when I click on the nitroshare indicator in the system tray, as soon as I move the pointer to select an item in nitroshare's popup menu - and therefore move the pointer off the panel - the panel minimizes, and nitroshare's popup menu disappears.  

Does my description make sense?  

Can this be fixed?  

Thanks.

**Comment by nathan-osman** [comment]
I'm going to install LXDE on one of my virtual machines to see if I can reproduce this.

**Comment by nathan-osman** [comment]
Yes, I was able to reproduce the exact behavior you described. I'm not sure how simple this would be to fix, but there is a workaround you can use in the meantime. NitroShare uses a system tray icon by default on LXDE but it can be "tricked" into using an AppIndicator.

Make sure NitroShare is not running and press <kbd>Alt</kbd>+<kbd>F2</kbd>. In the box that appears, paste the following command:

    env XDG_CURRENT_DESKTOP=unity nitroshare

This will launch NitroShare with the `XDG_CURRENT_DESKTOP` environment variable set to `unity`, which will cause the program to display an AppIndicator instead of a system tray icon.
